src/main/resources/templates/task.html

```diff
@@ -18,9 +18,9 @@
     <div>
         Status:
         <select th:field="${taskRequest.status}">
-            <option value="To do">To Do</option>
-            <option value="In Review">In Review</option>
-            <option value="In Progress">In Progress</option>
+            <option value="To do">To do</option>
```

Comment:
Wcześniej tego nie zauważyłem :) zastosować camel case dla wartości value np"toDo" zamiast "To do - spacje są dozwolone tylko w przypadku gdy jest to tekst, który bezpośrednio widzi użytkownik. Wszystko co jest "pod spodem" ma określone konwencje nazywnictwa ale nigdy nie używamy spacji